app/src/lib/editors/win32.ts

```diff
@@ -672,6 +721,7 @@ export async function getAvailableEditors(): Promise<
     results.push({
       editor: ExternalEditor.SlickEdit,
       path: slickeditPath,
+      usesShell: false,
```

Comment:
Can you provide some more context for this change? Has something changed in SlickEdit 2019 that necessitates the change and if so have we tested it for the previous versions of SlickEdit?

Reply:
This line was added for all the other editors but not SlickEdit, so I added it to be consistent with the others.

Reply:
Looking into this it seems the default for this option is false so this change is great for consistency and doesn't change the logic.